docs/install_guides/installing-openvino-apt.md

```diff
@@ -8,180 +8,3 @@ Currently only the following ways are provided to install OpenVINO™:
 * [Build From Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
 
 The other installation methods are temporarily unavailable.
-
-## System Requirements
-
-The complete list of supported hardware is available in the [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/openvino-relnotes.html).
```

Comment:
I suppose we should not see these changes here..

Reply:
That's true, an extra line slipped in when resolving merge conflicts. Fixed.